client/landing/stepper/declarative-flow/internals/steps-repository/intro/videopress-intro-modal-content.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import './videopress-intro-modal-styles.scss';
+import CheckmarkIcon from './icons/checkmark-icon';
+import { IntroModalContentProps } from './intro';
+
+const VideoPressIntroModalContent: React.FC< IntroModalContentProps > = ( { onSubmit } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="videopress-intro-modal">
+			<h1 className="intro__title">{ translate( 'Your video site, with no hassle' ) }</h1>
+			<div className="intro__description">
+				{ translate(
+					'Create a WordPress.com site with everything you need to share your videos with the world.'
+				) }
+			</div>
+			<ul className="videopress-intro-modal__list">
+				<li>
+					<CheckmarkIcon />
+					<span>
+						{ translate( '{{a}}Videomaker{{/a}}, a premium theme optimized to display videos.', {
+							components: {
+								a: (
+									<a
+										href="https://jetpack.com/features/design/themes/showcase/videomaker/"
+										target="_blank"
+										rel="external noreferrer noopener"
+									/>
+								),
+							},
+						} ) }
+					</span>
+				</li>
+				<li>
+					<CheckmarkIcon />
+					<span>
+						{ translate( 'Unbranded, ad-free, customizable {{a}}VideoPress{{/a}} player.', {
+							components: {
+								a: (
+									<a
+										href="https://videopress.com"
+										target="_blank"
+										rel="external noreferrer noopener"
+									/>
+								),
+							},
+						} ) }
+					</span>
+				</li>
+				<li>
+					<CheckmarkIcon />
+					<span>
+						{ translate( 'Upload videos directly to your site using the WordPress editor.' ) }
+					</span>
+				</li>
+				<li>
+					<CheckmarkIcon />
+					<span>{ translate( 'Up to 200GB of storage and your own domain for a year.' ) }</span>
+				</li>
+			</ul>
+			<div className="videopress-intro-modal__button-column">
+				<Button className="intro__button" primary onClick={ onSubmit }>
+					{ translate( 'Get Started' ) }
+				</Button>
+				<Button plain className="learn-more">
+					{ translate( 'Or learn more about VideoPress.' ) }
```

Comment:
This button doesn't do anything, is it meant to link to VideoPress.com?

Reply:
I thought I did at one point, maybe didn't commit it 😅